src/platform/nrfconnect/wifi/WiFiManager.cpp

```diff
@@ -433,5 +449,53 @@ void WiFiManager::PostConnectivityStatusChange(ConnectivityChange changeType)
     PlatformMgr().PostEventOrDie(&networkEvent);
 }
 
+#if CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_ENABLE
+System::Clock::Milliseconds32 WiFiManager::GetNextRecoveryTime()
+{
+    if (mConnectionRecoveryTimeMs >= kConnectionRecoveryMaxIntervalMs)
+    {
+        uint32_t jitter =
+            (jitter < kConnectionRecoveryMaxIntervalMs) ? jitter : static_cast<uint16_t>(kConnectionRecoveryMaxIntervalMs);
```

Comment:
Something is wrong here, I think we should use `kConnectionRecoveryJitterMs` in this algorithm.

Reply:
Yes, I see a fault. I will fix it.